applications/luci-app-babeld/htdocs/luci-static/resources/babeld.js

```diff
@@ -0,0 +1,171 @@
+function renderTableXRoutes(ubus_data, target_id) {
+	var data = JSON.parse(ubus_data);
+	for (var protocol in data) {
+		var target = document.getElementById(target_id);
+
+		var title = document.createElement('h3');
+		title.appendChild(document.createTextNode('X-Routes ' + protocol));
+		target.appendChild(title);
+
+		var table = document.createElement('table');
+		table.setAttribute('class', 'table');
+		table.setAttribute('id', 'babel_overview_xroutes_' + protocol);
+
+		var headerRow = document.createElement('tr');
+		headerRow.setAttribute('class', 'tr table-titles');
+		var headerContent = '<th class="th" style="font-weight: 700;">' + protocol + ' Prefix</th>\
+                             <th class="th" style="font-weight: 700;">Metric</th>\
+                             <th class="th" style="font-weight: 700;">Source-Prefix</th>';
+
+		headerRow.innerHTML = headerContent;
+		table.appendChild(headerRow);
+
+
+		for (var prefix in data[protocol]) {
+			var prefixRow = document.createElement('tr');
+			prefixRow.setAttribute('class', 'tr');
+			var prefixContent = '<td class="td" data-title="xroutes_' + protocol + '_prefix">' + prefix + '</td>\
```

Comment:
dito

Reply:
![image](https://github.com/openwrt/luci/assets/91346165/779d7268-38da-46f3-9a30-3276404d10d2)
I fixed all the issues and also tested it.